src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -338,6 +346,43 @@ public function callableMethodCommand(InputInterface $input, OutputInterface $ou
     {
         $output->writeln('from the code...');
     }
+
+    public function testSetProcessTitle()
+    {
+        $command = new TestCommand();
+
+        $this->assertSame('', cli_get_process_title());
+
+        $command->setProcessTitle('process title');
+        $inputMock = $this->prophesize(InputInterface::class);
+        $outputMock = $this->prophesize(OutputInterface::class);
+        $command->run($inputMock->reveal(), $outputMock->reveal());
+
+        $this->assertSame('process title', cli_get_process_title());
+    }
+
+    public function testSetAliases()
+    {
+        $command = new TestCommand();
+        $command->setAliases([]);
+        $command->setAliases(new TraversableDummy());
```

Comment:
At the moment,the test pass when `setAliases` is just an empty method with no arguments. Instead of testing that some syntax is correct, the behaviour/public API should be tested.

So in this case, verify that the aliases method is indeed setting the aliases, like:
```php
/**
 * @dataProvider getSetAliasesData
 */
public function testSetAliases($aliases, $expected = null)
{
    $command = new TestCommand();
    $command->setAliases($aliases);

    $this->assertEquals(null === $expected ? $aliases : $expected, $command->getAliases());
}

public function getSetAliasesData()
{
    return array(
        array(array('some:alias')),
        array(new \ArrayIterator(array('some:alias')), array('some:alias')),
    );
}

/**
 * @expectedException \Symfony\Component\Console\Exception\InvalidArgumentException
 */
public function testSetAliasesWithNonTraversable()
{
    $command = new TestCommand();
    $command->setAliases('some:alias');
}
```